weaveworks/weave-gitops-enterprise
# Fixes gitlab oauth redirection when creating-cluster

- We accidentally changed the query-string parsing lib
- `qs` was still available as sub-dep and so didn't break the build
  - Can we fix this? This is something `pnpm` protects against..

Introd' in https://github.com/weaveworks/weave-gitops-enterprise/pull/1670

### To test

```bash
cd ui-cra
PROXY_HOST=https://demo-01.wge.dev.weave.works/ yarn start
```

- http://localhost:3000/templates/capa-cluster-template/create
- Click auth-with-gitlab
- See the oauth flow successfully complete
